manifests/backend.pp

```diff
@@ -40,6 +40,10 @@
 #   This parameter is mutually exclusive with ssl_key_source
 # @param include_default_resources
 #   Sets if default sensu resources should be included
+# @param manage_agent_user
```

Comment:
Link the docs where Sensu says you should do this. Curious if this is a common use case.

Reply:
I think more appropriate to not use `manage_agent_user` and just disable the agent, so will update the docs for that parameter.